[uvw] build failure

**Host Environment**
- OS: Windows
- Compiler: revision

**To Reproduce**
'vcpkg.json' with uvw dependency.

**Failure logs**

> 
> 1> [CMake] CMake Error at C:/dev/vcpkg/scripts/buildsystems/vcpkg.cmake:824 (_find_package):
> 1> [CMake] Could not find a package configuration file provided by "unofficial-libuv"
> 1> [CMake] with any of the following names:
> 1> [CMake]
> 1> [CMake] unofficial-libuvConfig.cmake
> 1> [CMake] unofficial-libuv-config.cmake
> 1> [CMake]
> 1> [CMake] Add the installation prefix of "unofficial-libuv" to CMAKE_PREFIX_PATH or
> 1> [CMake] set "unofficial-libuv_DIR" to a directory containing one of the above
> 1> [CMake] files. If "unofficial-libuv" provides a separate development package or
> 1> [CMake] SDK, be sure it has been installed.
> 1> [CMake] Call Stack (most recent call first):
> 1> [CMake] out/build/windows-msvc-debug-hard-mode/vcpkg_installed/x64-windows/share/uvw/uvw-config.cmake:1 (find_package)
> 1> [CMake] C:/dev/vcpkg/scripts/buildsystems/vcpkg.cmake:824 (_find_package)
> 1> [CMake] knuckles/CMakeLists.txt:13 (find_package)

**Additional context**

hi guys i got this error from 'uvw' in vcpkg.
i'm using 'vcpkg.json' and constrained version by specifying builtin-baseline

"builtin-baseline": "98f8d00e89fb6a8019c2045cfa1edbe9d92d3405"

if there are anything that i need to modify to make it work please let me know about it.

https://github.com/microsoft/vcpkg/issues/25563
https://github.com/microsoft/vcpkg/issues/24743
https://github.com/microsoft/vcpkg/pull/24745

--- a/ports/uvw/uvw-config.cmake
+++ b/ports/uvw/uvw-config.cmake
@@ -1,2 +1,2 @@
-find_package(unofficial-libuv CONFIG REQUIRED)
+find_package(libuv CONFIG REQUIRED)
 include ("${CMAKE_CURRENT_LIST_DIR}/uvw-targets.cmake")

--- a/ports/uvw/vcpkg.json
+++ b/ports/uvw/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "uvw",
   "version": "2.12.1",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Header-only, event based, tiny and easy to use libuv wrapper in modern C++.",
   "homepage": "https://github.com/skypjack/uvw",
   "license": "MIT",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -7358,7 +7358,7 @@
     },
     "uvw": {
       "baseline": "2.12.1",
-      "port-version": 1
+      "port-version": 2
     },
     "uwebsockets": {
       "baseline": "20.11.0",

--- a/versions/u-/uvw.json
+++ b/versions/u-/uvw.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "c36c66a8d5f4a6eed4aa9fe4399f32a516d3883c",
+      "version": "2.12.1",
+      "port-version": 2
+    },
+    {
       "git-tree": "609b48de110aa8763d8f4a674ac3aa7c5821382f",
       "version": "2.12.1",
       "port-version": 1

